customLintAAR/src/test/java/com/zulip/android/customlintarr/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package com.zulip.android.customlintarr;
```

Comment:
unless this file is used to test the custom lint rule, I think it can be deleted.